hidimstat/test/test_desparsified_lasso.py

```diff
@@ -4,29 +4,73 @@
 
 import numpy as np
 from numpy.testing import assert_almost_equal
+from scipy.linalg import toeplitz
 
-from hidimstat.desparsified_lasso import desparsified_lasso_confint
+from hidimstat.scenario import multivariate_1D_simulation
+from hidimstat.scenario import multivariate_temporal_simulation
+from hidimstat.desparsified_lasso import desparsified_lasso
+from hidimstat.desparsified_lasso import desparsified_group_lasso
 
 
-def test_desparsified_lasso_confint():
+def test_desparsified_lasso():
```

Comment:
you should have short docstrings (maybe just one line) for each test to say
more specifically what you test.

Reply:
Ok I will do that in another PR.